### ISSUE TITLE
Cannot set font sizes for Text/TextArea in UI Form

### ISSUE THREAD

**Issue by gamename** [aws-amplify/amplify-ui#4252]
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have tried disabling all browser extensions or using a different browser
- [X] I have tried deleting the node_modules folder and reinstalling my dependencies
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

React

### Which UI component?

Primitive components

### How is your app built?

Next.js

### What browsers are you seeing the problem on?

Chrome, Firefox, Safari

### Which region are you seeing the problem in?

us-west-1

### Please describe your bug.

I'm not able to override the font sizes in text and textarea fields for a Form in the UI Library.  I *am* able to override the font sizes for the labels, however.  

### What's the expected behaviour?

I should be able to override the text/textarea fields to match the fontsizes I want. 

### Help us reproduce the bug!

Use [this](https://gist.github.com/gamename/33f275f20bfef2ae49089b47a1743465) snippet to call [this](https://gist.github.com/gamename/e8eb66fc4a62b9219efa4019ec08ae91) component. 

### Code Snippet

see above

### Console log output

No console log output generated

### Additional information and screenshots

![image](https://github.com/aws-amplify/amplify-ui/assets/42153/bb64370d-2e97-48f0-b7d2-fdc0bd285e05)

**Comment by gamename** [comment]
Found a work-around.

**Comment by OperationalFallacy** [comment]
@gamename, what was your workaround? I'm looking to do these customizations, too

**Comment by gamename** [comment]
I basically built the component by hand. That was my workaround.

**Comment by OperationalFallacy** [comment]
Interesting. Did you use any functions or types the auto-generated form provides?

I don't get why properties work randomly, like `fontStyle` or `fontWeight` but not `fontSize` 
<UserCreateForm width="80%" fontSize="small" fontWeight={'100'} fontStyle={'italic'}/>

**Comment by gamename** [comment]
@OperationalFallacy I agree.  For some reason, the attributes of the field itself cannot (that i know of) be changed.

**Comment by OperationalFallacy** [comment]
I managed to change fontsize on the buttons using tokens. Maybe there is a component for these fields? 

```
  tokens: {
    components: {
      button: {
        fontSize: {
          value: "0.7rem",
        },
      },
    }
    ```

**Comment by gamename** [comment]
@OperationalFallacy I explored that, but didn't find anything.  Maybe you'll have better luck. :) Let me know if you do.  I could still use the info.
